features/audio-playback/README.md

```diff
@@ -0,0 +1,2 @@
+Audio sample: THROW THE SWITCH by Robert Meyers
```

Comment:
Sorry for nitpicking but it takes a lot of time for the modified sample to start playing. I'm guessing that it'd be faster if we used a shorter input sample (current one is 6min)?